### PR TITLE
Add help for common IQ# installation issue

### PR DESCRIPTION
**Pull request by rmshaffer**
This PR adds some troubleshooting steps for a common IQ# installation issue (see https://github.com/microsoft/iqsharp/issues/14).

Here's how the change looks when rendered: 
image

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/26367a8577b74560608de8fc8cd3945a5d0335c2/articles/install-guide/pyinstall.md) | 95 |   | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a79cc6c8-195d-4726-b5f4-7497ace92e3a_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/qjupyter.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/26367a8577b74560608de8fc8cd3945a5d0335c2/articles/install-guide/qjupyter.md) | 88 |   | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ba02a9c4-817c-47cc-baef-a3de771419cc_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/install-guide/pyinstall.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ Install the QDK to develop Python host programs to call Q# operations.
     dotnet tool install -g Microsoft.Quantum.IQSharp
     dotnet iqsharp install
     ```
+
+    > [!NOTE]
+    > If you get an error during the `dotnet iqsharp install` step, open a new terminal window and try again.
+    > If this still doesn't work, try locating the installed `dotnet-iqsharp` tool (on Windows, `dotnet-iqsharp.exe`) and running:
+    > ```
+    > /path/to/dotnet-iqsharp --user --path-to-tool="/path/to/dotnet-iqsharp"
```
Comment (cgranade) [inline articles/install-guide/pyinstall.md:39]:
```suggestion
    > /path/to/dotnet-iqsharp install --user --path-to-tool="/path/to/dotnet-iqsharp"
```

File: articles/install-guide/qjupyter.md [hunk 2]
```diff
@@ -32,6 +32,14 @@ IQ# (pronounced i-q-sharp) is an extension primarily used by Jupyter and Python
     dotnet iqsharp install
     ```
 
+    > [!NOTE]
+    > If you get an error during the `dotnet iqsharp install` step, open a new terminal window and try again.
+    > If this still doesn't work, try locating the installed `dotnet-iqsharp` tool (on Windows, `dotnet-iqsharp.exe`) and running:
+    > ```
+    > /path/to/dotnet-iqsharp --user --path-to-tool="/path/to/dotnet-iqsharp"
```
Comment (cgranade) [inline articles/install-guide/qjupyter.md:39]:
```suggestion
    > /path/to/dotnet-iqsharp install --user --path-to-tool="/path/to/dotnet-iqsharp"
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/cd1a49a15f65ccecca0e547e884c5e6344c2f049/articles/install-guide/pyinstall.md) | 95 |   | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4d5c25c6-a3ed-48dc-855c-59e532fef0cc_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/qjupyter.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/cd1a49a15f65ccecca0e547e884c5e6344c2f049/articles/install-guide/qjupyter.md) | 88 |   | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ba397886-0ade-4f12-81dc-e70b4ef2ab8d_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/install-guide/pyinstall.md (hunk 1 above)
Comment (rmshaffer) [reply articles/install-guide/pyinstall.md:39]: sigh, thank you 👍

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2b30c39f6ac135d3654e770c3a6f298a1b6463ea/articles/install-guide/pyinstall.md) | 95 |   | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ec2d194b-82fb-499d-aebf-7e4d937b70b6_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/qjupyter.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2b30c39f6ac135d3654e770c3a6f298a1b6463ea/articles/install-guide/qjupyter.md) | 88 |   | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/353acc98-4e7b-4817-b2f7-cbcbf9dcb24a_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2b30c39f6ac135d3654e770c3a6f298a1b6463ea/articles/install-guide/pyinstall.md) | 95 |   | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c6dc064e-75b6-498a-a4ad-e7d04e5c0291_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/qjupyter.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/2b30c39f6ac135d3654e770c3a6f298a1b6463ea/articles/install-guide/qjupyter.md) | 88 |   | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/edb6364e-eed2-4609-b9c5-a9bdc572a17c_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by rmshaffer** [comment]
> Honestly, this could even be a pretty reasonable microsoft/jupyter-core feature; perhaps a new `--find-path` option that tries to guess the path from `System.Reflection.Assembly.GetExecutingAssembly().Location`?

@cgranade agreed, that would be nice. Opened https://github.com/microsoft/jupyter-core/issues/56.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4c20e8ff228e6ab71c1577dfe4240e7f841d3dc7/articles/install-guide/pyinstall.md) | 95 |   | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b64e65f1-7c29-4287-ab0e-df77057fef32_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/qjupyter.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4c20e8ff228e6ab71c1577dfe4240e7f841d3dc7/articles/install-guide/qjupyter.md) | 88 |   | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c4763a23-82a6-4334-bed0-dbc630a960ed_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/pyinstall.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/bfe59861224a1a7b1742ede789fb5b91cc15fe00/articles/install-guide/pyinstall.md) | 95 |   | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/36a1342d-4eff-4c35-b3fb-5910858b1bb0_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/install-guide/qjupyter.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/bfe59861224a1a7b1742ede789fb5b91cc15fe00/articles/install-guide/qjupyter.md) | 88 |   | 0 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/de9e23f4-09d2-40d9-9296-3378182d59fb_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**